platform/mellanox/install-pending-fw.py

```diff
@@ -45,7 +45,7 @@
                 print("WARNING: Multiple CPLD firmwares defined. Some CPLD updates may not fully complete.")
             CPLD_FLAG = (component, fw_file)
         else:
-            component.install_firmware(fw_file)
+            component.__install_firmware(fw_file, allow_reboot=False)
```

Comment:
@alexrallen in python double underscore notation `__` is for private calls only. You can't use it for public APIs.

Reply:
How do you suggest I handle this?

Reply:
@alexrallen use notation without underscore and define a separate API

Reply:
Sure.